AspNetCore.sln

```diff
@@ -1379,7 +1379,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "http2cat", "src\Servers\Kes
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Transport.Quic", "Transport.Quic", "{EE9D0952-6060-4723-B329-94A2950A6762}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Server.Kestrel.Transport.Experimental.Quic", "src\Servers\Kestrel\Transport.Quic\src\Microsoft.AspNetCore.Server.Kestrel.Transport.Experimental.Quic.csproj", "{132D43A2-067A-4E24-A520-45B9F14DCB8E}"
```

Comment:
Why are we removing Experimental from the namespace right now?

Reply:
Yeah this was the main question. If our intention is to ship this in 6.0, we'd want to remove the experimental label no matter what. IMO I'd rather remove experimental in 6.0 now, and in emergency readd it. Also, anyone who is depending on these won't be broken when we swap namespaces.

Reply:
I think we should have a proper API review before we move this into a proper namespace. Ideally, we would do that before even making this public again.

I don't want to end up in a situation where we ship that hasn't been properly vetted because of inertia. Leaving "Experimental" in the namespace for now will encourage us to at least give this another pass before shipping. I'm not too concerned about breaking people using this in a preview. We shouldn't feel like we cannot make breaking changes to this API because it will break code depending on previews. Removing ".Experimental" from some `using`s is easy enough if that's all we decide to do.

Reply:
Alright, that's fine with me for now.